test/load/packer/README.md

```diff
@@ -0,0 +1,22 @@
+## Quick start
+
+# Consul AMI:
+
+Within the `consul-ami/` directory
+1) Retrieve your [Datadog API key]((https://docs.datadoghq.com/account_management/api-app-keys/#api-keys)), set this as an environment variable, ex: `export DD_API_KEY=$YOURDDAPIKEYHERE`
+2) Set the AWS_DEFAULT_REGION for Packer, ex: `export AWS_DEFAULT_REGION=us-east-1`
+3) Run `packer build consul.json`. 
+
+When the build finishes, it will output the IDs of the new AMI. Save those AMI IDs as they will be used later as variables needed for Terraform! 
```

Comment:
Perhaps this point could be moved to the opening section and reworded so it's clear that all AMI IDs need to be saved for later use.

Reply:
got it!